src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -8697,6 +8697,25 @@ void M(bool x)
             await VerifyItemExistsAsync(markup, "x");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ExceptionFilter1_NotBeforeOpenParen()
```

Comment:
just a precaution to make sure this never gets mixed up with `when` inside switch where there's no parentheses required